services/graph/pkg/service/v0/driveitems.go

```diff
@@ -181,8 +181,9 @@ func cs3ResourceToRemoteItem(res *storageprovider.ResourceInfo) (*libregraph.Rem
 		Size: size,
 	}
 
-	if name := path.Base(res.Path); name != "" {
+	if name := path.Base(res.GetPath()); name != "" {
 		remoteItem.Name = &name
+		remoteItem.Path = libregraph.PtrString(utils.MakeRelativePath(res.GetPath()))
```

Comment:
This results in paths with a prefix of `./`. Could you strip that away? I.e. instead of `./test from admin/test` I'd like to have `test from admin/test` in the `path` attribute.

Reply:
If I don't make a relative path you would get a path with a leading slash. Would that be ok?

I could also already append the path to the webdav base URL to have a ready-to-use webDav Url.